pkg/aws/policies.go

```diff
@@ -1260,12 +1267,18 @@ func (c *awsClient) GetAttachedPolicy(role *string) ([]PolicyDetail, error) {
 	}
 
 	for _, policy := range attachedPoliciesOutput.AttachedPolicies {
-		policyDetail := PolicyDetail{
-			PolicyName: aws.ToString(policy.PolicyName),
-			PolicyArn:  aws.ToString(policy.PolicyArn),
-			PolicyType: Attached,
+		tagFit, err := isPolicyTagFit(c.iamClient, policy.PolicyArn, tagFilter)
```

Comment:
NP: Could we maybe rename `tagfit`? Not sure I understand what it does/is without looking at the function

Reply:
thanks for point this out, I updated the name for the var and function. Is current one better?